packages/elements/package.json

```diff
@@ -40,7 +49,9 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "files": [
-    "dist"
+    "dist",
+    "!dist/metafile-*.json",
```

Comment:
Generated for bundle analysis and unable to configure where they're generated. This ensures we don't include them in the resulting package.